persistence-modules/spring-data-jpa-repo-3/src/main/java/com/baeldung/spring/data/jpa/naturalid/repository/GuestRoomJpaRepo.java

```diff
@@ -0,0 +1,9 @@
+package com.baeldung.spring.data.jpa.naturalid.repository;
+
+import com.baeldung.spring.data.jpa.naturalid.entity.GuestRoom;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface GuestRoomJpaRepo extends JpaRepository<GuestRoom, Long> {
```

Comment:
Let's be consistent in our repository naming -- i.e., don't shorten to "Repo".

Reply:
renamed